.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -5,31 +5,61 @@ about: Report a bug with this project. Make sure you've read "Things To Know Bef
 
 ---
 
-### Things That I, The Bug Reporter, Have Done Before Submitting
-- [ ] I have ensured that this is a bug. Where I was not sure, I checked the [forum](https://forum.pi-top.com/c/pi-top-software), and opened a discussion there.
-- [ ] I have read [Things To Know Before Submitting A Bug Report](https://github.com/pi-top/pi-top-Python-SDK/wiki/Things-To-Know-Before-Submitting-A-Bug-Report) before filling this out.
+<!--
+  ✏️ Fill out the following document, following the template provided by the
+  headings and these comments.
```

Comment:
People might not know this is a comment, so we can probably do a bit more to clarify on that